pkg/controller/controller_binding.go

```diff
@@ -351,6 +351,13 @@ func (c *controller) reconcileServiceBindingDelete(binding *v1beta1.ServiceBindi
 
 	instance, err := c.instanceLister.ServiceInstances(binding.Namespace).Get(binding.Spec.ServiceInstanceRef.Name)
 	if err != nil {
+		if apierrors.IsNotFound(err) {
+			glog.V(5).Info(pcb.Messagef(`References a non-existent %s "%s/%s"`,
+				pretty.ServiceInstance, binding.Namespace, binding.Spec.ServiceInstanceRef.Name,
+			))
+			return c.processUnbindSuccess(binding)
+		}
+
 		msg := fmt.Sprintf(
```

Comment:
Is this message still applicable now that we are explicitly handling a not found above? It seems that once you fall through to this point, the error wasn't a non-existent instance, but instead some other unspecified plumbing error that caused the retrieve to fail.

Reply:
I don't think the message was ever fully correct. I'm not familiar with the errors returned by a client.

WDYT about change it to something like `error getting the` and try to add the err to the message? @jberkhahn

Reply:
👍

Reply:
sure, seems good